cannot build with version 0.16.0 or higher on gradle 4.10

With Version 0.15.0 and gradle 4.10, everything works as expected.
When updating to 0.16.0 or 0.17.0 I am getting following output:

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':app'.
> Failed to notify project evaluation listener.
   > org.gradle.api.tasks.TaskContainer.named(Ljava/lang/String;Ljava/lang/Class;Lorg/gradle/api/Action;)Lorg/gradle/api/tasks/TaskProvider;
```

The solution is quite easy, just update gradle. But it's nowhere mentioned.

Tested with gradle 5.2.1

--- a/README.md
+++ b/README.md
@@ -2,6 +2,8 @@
 
 Gradle plugin that configures [Error Prone](http://errorprone.info/), [Findbugs](http://findbugs.sourceforge.net/), [Checkstyle](http://checkstyle.sourceforge.net/), [PMD](https://pmd.github.io/), [CPD](https://pmd.github.io/pmd-6.0.0/#cpd), [Lint](https://developer.android.com/studio/write/lint.html), [Detekt](https://github.com/arturbosch/detekt) & [Ktlint](https://github.com/shyiko/ktlint). All of these tools are also automatically hooked into the `check` gradle task. Below, I'll go more into depth how each of those plugins are configured.
 
+This plugin requires Gradle 5.0 or later.
+
 # Set up
 
 **root/build.gradle**

